internal/build/local_docker.go

```diff
@@ -284,6 +289,26 @@ func tarContextAndUpdateDf(ctx context.Context, df Dockerfile, paths []pathMappi
 func readDockerOutput(ctx context.Context, reader io.Reader) (*json.RawMessage, error) {
 	span, ctx := opentracing.StartSpanFromContext(ctx, "daemon-readDockerOutput")
 	defer span.Finish()
+
+	displayCh := make(chan *client.SolveStatus)
+
+	displayStatus := func(displayCh chan *client.SolveStatus) {
+		out := os.Stdout
+		c, err := console.ConsoleFromFile(out)
```

Comment:
it might be worthwhile to pair on how to use dependency injection to get a console, rather than hard-coding os.Stdout here

Reply:
I'd love that! maybe tomorrow morning?